src/frontend/src/common/redux/actions/index.ts

```diff
@@ -0,0 +1,15 @@
+// export action type for use in reducers and middleware
+// persisted names are for use in localstorage
+export const groupPersistedName = "currentGroup";
+export const setGroupActionType = "group/setGroup";
```

Comment:
I have a strong belief that we should have a dedicated `actionTypes` enum/var -- I'd lean to it being its own dedicated file, but I'm not against it living with the actions (especially while we have so few types right now) -- rather than independent actionType variables.

If I interrogate my reasoning, a good chunk of it is that it's just what I'm used to from prior projects, but I do think it's useful to have a canonical location for them. Also it helps prevent the temptation to just directly use a string in your action/reducer. Also also, if we have an enum of them, we can then require the ActionType TS type to have its `type` property be one of those values, which seems like a good safety feature.